chips/lowrisc/Cargo.toml

```diff
@@ -7,3 +7,7 @@ edition = "2021"
 [dependencies]
 rv32i = { path = "../../arch/rv32i" }
 kernel = { path = "../../kernel" }
+
+[target.'cfg(test)'.dependencies.kernel]
```

Comment:
Is there a reason you used `[target.'cfg(test)'.dependencies]` rather than `[dev-dependencies]`?

Reply:
I had convinced myself that `[dev-dependencies]` didn't do what I needed, but it seems that it does.  Fog of war.